### PR TITLE
Check for unique ID columns in origins/destinations

### PR DESCRIPTION
**Pull request by christophfink**
Continuing here with what was begun in PR #140 and #165

### CODE REVIEW

**Comment by christophfink** [comment]
closes #161 
closes #162 
closes #140

File: src/r5py/util/check_od_data_set.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```
Comment (HTenkanen) [inline src/r5py/util/check_od_data_set.py:1]: Might we also have other "checks" for files/inputs etc? Just thinking that should we have a more generic filename into which we could possibly add also other checks in the future. The file could e.g. be `validate_input.py` or something along those lines..

**Review by HTenkanen — APPROVED** [review]
One comment regarding the name of the file for you to consider, but that is not really critical.

File: src/r5py/util/check_od_data_set.py (hunk 1 above)
Comment (christophfink) [reply src/r5py/util/check_od_data_set.py:1]: that’s true :+1:

Comment (christophfink) [reply src/r5py/util/check_od_data_set.py:1]: I went even bigger and called the file `data_validation.py`

**Pull request merged**